lib/serialize.js

```diff
@@ -0,0 +1,230 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as base58 from 'base58-universal';
+import * as base64url from 'base64url-universal';
+import {
+  ALGORITHM,
+  ECDSA_CURVE,
+  EXTRACTABLE,
+  MULTIBASE_BASE58_HEADER,
+  MULTIKEY_CONTEXT_V1_URL
+} from './constants.js';
+import {webcrypto} from './crypto.js';
+import {
+  getNamedCurve,
+  getSecretKeySize,
+  setPublicKeyHeader,
+  setSecretKeyHeader
+} from './helpers.js';
+
+// FIXME: may need to move any leading zeros for bitstring compression; needs
```

Comment:
We will need to address this FIXME before releasing v1. This should be completed in a separate PR.

cc: @dlongley

Reply:
Created Issue #2 to address this `FIXME` comment